### PR TITLE
Agile talk

### PR DESCRIPTION
**Pull request by ChrisBeeley**

### CODE REVIEW

File: presentations/2024-08-22_agile-and-scrum/index.qmd [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+## More agile advantages
+
+* Early and cheap failure
+* Continous testing and QA
+* Reduction in unproductive work
+* Team can improve regularly, not just the product
+
+## Agile methods
+
+* There are lots of agile methdologies
```
Comment (matt-dray) [inline presentations/2024-08-22_agile-and-scrum/index.qmd:108]:
```suggestion
* There are lots of agile methodologies
```

File: presentations/2024-08-22_agile-and-scrum/index.qmd [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+## Regular stakeholder feedback
+
+* Agile teams are very responsive to product feedback 
+* The project we're curently working on is **very** agile whether we like it or not
+* Our customers never know what they want until we show them something they **don't** want
+
+## More agile advantages
+
+* Early and cheap failure
+* Continous testing and QA
```
Comment (matt-dray) [inline presentations/2024-08-22_agile-and-scrum/index.qmd:102]:
```suggestion
* Continuous testing and QA
```

**Review by matt-dray — APPROVED** [review]
I've done a rapid check of the text content ahead of the talk. Approving, but suggested typo fixes are below.

File: presentations/2024-08-22_agile-and-scrum/index.qmd [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+* Scrum is the agile methodology we have adopted
+* Despite dire warnings to the contrary we have not adopted it wholesale but most of its principles
+* The fundamental organising principle of work in scrum is a **sprint** lasting 1-4 weeks
+* Each sprint finishes with a defined and useful piece of software that can be shown to/ used by customers
+
+## Product owner
+
+* This person is responsible for the backlog- what goes in to the sprint
+* The backlog should be inclusive of all of the things that customers want or might want
+* They backlog should be prioritised
```
Comment (matt-dray) [inline presentations/2024-08-22_agile-and-scrum/index.qmd:123]:
```suggestion
* The backlog should be prioritised
```

**Review by StatsRhian — APPROVED** [review]

**Comment by ChrisBeeley** [comment]
Thanks Matt!

**Pull request merged**